src/fileformats/iof_course_export.cpp

```diff
@@ -97,11 +99,35 @@ void IofCourseExport::writeXml(const PathObject& object)
 	}
 	{
 		XmlElementWriter event(*xml, QLatin1String("RaceCourseData"));
+		writeMap();
 		writeControls(object.getRawCoordinateVector());
 		writeCourse(object.getRawCoordinateVector());
 	}
 }
 
+void IofCourseExport::writeMap()
+{
+	XmlElementWriter map_data(*xml, QLatin1String("Map"));
+	xml->writeTextElement(QLatin1String("Scale"), QString::number(map->getScaleDenominator()));
+	
+	// IOF.xsd proposes for 'MapPositionTopLeft': "The position of the map's top left corner given in the map's coordinate system, usually (0, 0)."
+	// However, the real coordinates from the map_extent structure are used below.
+	// NOTE: templates are considered when determining the map extent.
+	const auto map_extent = map->calculateExtent(true, true, view);
+	{
+		XmlElementWriter map_position_topleft(*xml, QLatin1String("MapPositionTopLeft"));
+		map_position_topleft.writeAttribute(QLatin1String("x"), QString::number(map_extent.topLeft().x()));
+		map_position_topleft.writeAttribute(QLatin1String("y"), QString::number(map_extent.topLeft().y()));
+		map_position_topleft.writeAttribute(QLatin1String("unit"), QLatin1String("mm"));	// optional attribute, default is 'mm'
+	}
+	{
+		XmlElementWriter map_position_bottomright(*xml, QLatin1String("MapPositionBottomRight"));
+		map_position_bottomright.writeAttribute(QLatin1String("x"), QString::number(map_extent.bottomRight().x()));
+		map_position_bottomright.writeAttribute(QLatin1String("y"), QString::number(map_extent.bottomRight().y()));
+		map_position_bottomright.writeAttribute(QLatin1String("unit"), QLatin1String("mm"));	// optional attribute, default is 'mm'
+	}
```

Comment:
Is the extent information useful at all when not also providing the `Image` element?

Reply:
> Is the extent information useful at all when not also providing the `Image` element?

Good question, but I assume yes, since the 'Image' element is not mandatory and the PR works for the issue reporter due to #2166